### PR TITLE
Added logging and updated language

### LINKED ISSUE #167: Display log values
It would be nice to display log messages not as warnings but maybe in a tab next to the arguments?

image

When a warning has warn_code 2, this is a log message. see: https://docs.thingsdb.net/v1/listening/warning/

### PR DESCRIPTION
**Pull request by AnjaBruls**
This resolves #167 and #169.

### CODE REVIEW

File: react/src/Stores/BaseStore.js [hunk 1]
```diff
@@ -286,7 +288,19 @@ class EventStore extends BaseStore {
         });
 
         socket.on('OnWarning', (data) => {
-            ErrorActions.setToastError(data.Msg);
+            if(data.Code === 2) {
```
Comment (joente) [inline react/src/Stores/BaseStore.js:291]: Maybe define `2` as a constant? You might also want to add a link to the docs to explain why `2`.

File: react/src/Components/Utils/QueryOutput.js [hunk 2]
```diff
@@ -30,6 +30,7 @@ const QueryOutput = ({output, onArgs, onChangeTab, tabIndex}) => {
                 <Tab label="Tree view" />
                 <Tab label="JSON view" />
                 {onArgs && <Tab label="Arguments" />}
+                {showLogs && <Tab label="Logging" />}
             </Tabs>
             <Collapse in={tabIndex === 0}>
```
Comment (joente) [inline react/src/Components/Utils/QueryOutput.js:35]: I think it is a good idea to define an enumerator for the tab indexes. Something like:
```javascript
const Tab = {
   TREE: 0,
   JSON: 1,
   ARGUMENTS: 2,
   LOG: 3
};
Object.freeze(Color);
```

File: react/src/Stores/BaseStore.js (hunk 1 above)
Comment (AnjaBruls) [reply react/src/Stores/BaseStore.js]: see d5c611e

File: react/src/Components/Utils/QueryOutput.js (hunk 2 above)
Comment (AnjaBruls) [reply react/src/Components/Utils/QueryOutput.js]: see d5c611e

**Review by joente — APPROVED** [review]

**Pull request merged**